doc/source/whatsnew/v1.2.0.rst

```diff
@@ -774,6 +774,7 @@ Other
 - Passing an array with 2 or more dimensions to the :class:`Series` constructor now raises the more specific ``ValueError`` rather than a bare ``Exception`` (:issue:`35744`)
 - Bug in ``dir`` where ``dir(obj)`` wouldn't show attributes defined on the instance for pandas objects (:issue:`37173`)
 - Bug in :meth:`RangeIndex.difference` returning :class:`Int64Index` in some cases where it should return :class:`RangeIndex` (:issue:`38028`)
+- Bug in :meth:`Index.intersection` returning duplicates when at least one of the indexes had duplicates (:issue:`31326`)
```

Comment:
need to remove this

Reply:
Sorry, probably messed up the merge. Is gone now